cmd/fluent-watcher/fluentbit/Dockerfile

```diff
@@ -1,13 +1,19 @@
-FROM golang:1.20.4-alpine3.16 as buildergo
+ARG FLUENTBIT_BASE
+FROM ghcr.io/verrazzano/verrazzano-base:v1.0.0-20230327155846-4653b27 as buildergo
+RUN microdnf update \
+    && microdnf install -y \
+    golang && \
+    microdnf clean all
+
 RUN mkdir -p /fluent-bit
 RUN mkdir -p /code
 COPY . /code/
 WORKDIR /code
 RUN echo $(ls -al /code)
 RUN CGO_ENABLED=0 go build -ldflags '-w -s' -o /fluent-bit/fluent-bit /code/cmd/fluent-watcher/fluentbit/main.go
 
-FROM fluent/fluent-bit:2.1.9
-LABEL Description="Fluent Bit docker image" Vendor="Fluent" Version="1.0"
+FROM $FLUENTBIT_BASE
+LABEL Description="Fluent Bit docker image" Vendor="Verrazzano" Version="2.1.10"
```

Comment:
Why do we have fluentbit version hard code mention here

Reply:
That's a label added by upstream, I mainly changed the vendor to Verrazzano and version to actual fluentbit version instead of "1.0".

Reply:
Ok